objects.md

```diff
@@ -0,0 +1,118 @@
+# About
+
+This repo needs to represent the following information:
+
+1. Adding/removing versions to channels
+2. Remove or add edges to all channels
```

Comment:
This happens at release-build time, because we want, but don't currently require, edge recommendations to be part of the signed release content (to guard against compromised Cincinnati).  Also, once we grow phased edge roll-outs, we're going to want per-channel start times and durations for those roll-outs (see also [here][1]).

[1]: https://github.com/openshift/cincinnati-graph-data/pull/1#discussion_r321911263

Reply:
adding happens at build time. yes.

however with the 'policy' object we would really like, we need some way to 're-add' back to what was in payload.